lib/asciitable/table.go

```diff
@@ -60,6 +63,62 @@ func MakeTable(headers []string) Table {
 	return t
 }
 
+func MakeTableWithRows(headers []string, rows [][]string) Table {
```

Comment:
Nit: Instead of making a separate function, what do you think about updating MakeTable to support accepting variable length argument with rows? Current invocations won't change and we'll have one less function.

```go
func MakeTable(headers []string, rows ...[]string) Table {
  // existing impl.
  for _, row := range rows {
    t.AddRow(row)
  }
  return t
}
```